sympy/physics/continuum_mechanics/beam.py

```diff
@@ -0,0 +1,567 @@
+"""
+This module can be used to solve 2D beam bending problems with
+singularity functions in mechanics.
+
+"""
+
+from __future__ import print_function, division
+
+from sympy.core import S, Symbol, Eq, diff
+from sympy.solvers import linsolve
+from sympy.printing import sstr
+from sympy.functions import SingularityFunction
+from sympy.core import sympify
+from sympy.integrals import integrate
+
+
+class Beam(object):
+    """
+    A Beam is a structural element that is capable of withstanding load
+    primarily by resisting against bending.
+    Beams are characterized by their profile (Second moment of area),
+    their length, and their material.
+
+    Parameters
```

Comment:
Move these into the `__init__` docstring. Only describe attributes and methods here at the class level. See the numpydoc examples for classes.